### PR TITLE
docs: adds import statement to info doc for elements

### PR DESCRIPTION
**Pull request by langz**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/elements/heading/info.mdx [hunk 1]
```diff
@@ -2,6 +2,13 @@
 showTabs: true
 ---
 
+## Import
+
+```tsx
+import { Heading } from '@dnb/eufemia/components'
+import { H1, H2, H3, H4, H5, H6 } from '@dnb/eufemia'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/elements/heading/info.mdx:9]:
```suggestion
import { H1, H2, H3, H4, H5, H6 } from '@dnb/eufemia/elements'
```

File: packages/dnb-design-system-portal/src/docs/uilib/elements/image/info.mdx [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
 
 The image element associated with the class `dnb-img` does not have much opinionated styling. It exists more to have a future possibility to optimize and add features.
 
 As of now, the React image "element" (Img) does provide a `figure` element with the `role="img"` and an `img` tag inside. This is mainly to support the [Skeleton](/uilib/components/skeleton) provider.
-
-```jsx
-import { Img } from '@dnb/eufemia'
-
-render(<InlineImg alt="..." src="..." width="100" height="100" />)
-```
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/elements/image/info.mdx:15]: This example was only to show the Import statement.

File: packages/dnb-design-system-portal/src/docs/uilib/elements/paragraph/demos.mdx [hunk 3]
```diff
@@ -7,10 +7,15 @@ import {
...
   ParagraphAdditional,
   ParagraphRegressionTests,
+  ParagraphModifiers,
 } from 'Docs/uilib/elements/paragraph/Examples'
 
 ## Demos
 
+### Paragraphs modifiers
+
+<ParagraphModifiers />
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/elements/paragraph/demos.mdx:17]: Moved this example to demos, away from info

**Review by tujoworker — APPROVED** [review]
Nice 🥇

**Pull request merged**